ccvm_simulators/ccvmplotlib/problem_metadata/boxqp_metadata.py

```diff
@@ -18,7 +19,11 @@ class for the BoxQP problem. This takes Box QP problem result data and
     generates plotting data.
     """
 
-    def __init__(self, problem: ProblemType, TTS_type: TTSType) -> None:
+    def __init__(
+        self,
+        problem: ProblemType,
+        TTS_type: TTSType,
```

Comment:
See my comment above about the use of `TTS_Type`.

Reply:
Fixed it in the main code.